src/patterns/dosdp-patterns/taxonSpecific.yaml

```diff
@@ -0,0 +1,38 @@
+pattern_name: taxonSpecific
+pattern_iri: http://purl.obolibrary.org/obo/cl/taxonSpecific
+
+description: A cell that is restricted to a specific taxon, such as CL:0001200 'lymphocyte of B lineage, CD19-positive' are only in mammals.
```

Comment:
It's important to be clear here. This is not to be used for any cell that is restricted to a taxon. This is for taxon-specific subclasses of existing cell types

We also want to note this should hardly ever be used